src/Microsoft.AspNet.Mvc/MvcServiceCollectionExtensions.cs

```diff
@@ -34,10 +34,77 @@ public static IServiceCollection AddMvc([NotNull] this IServiceCollection servic
         {
             ConfigureDefaultServices(services);
 
+            AddMvcServices(services);
+
+            return services;
+        }
+
+        /// <summary>
+        /// Configures a set of <see cref="MvcOptions"/> for the application.
+        /// </summary>
+        /// <param name="services">The services available in the application.</param>
+        /// <param name="setupAction">The <see cref="MvcOptions"/> which need to be configured.</param>
+        public static void ConfigureMvc(
```

Comment:
I'm missing something. `WithControllersAsServices()` still uses `TryAdd()` and none of these three methods have different visibility than before.

Another way of asking my question: The `AddMvcServices()` method has been refactored out of `AddMvc()` and changed (for 7 services) to use `Add()` instead of `TryAdd()`. Did the remainder of the file change only to place `public` methods above `internal` ones?